ui/lib/pki/addon/components/page/pki-issuer-edit.hbs

```diff
@@ -0,0 +1,77 @@
+<form {{on "submit" (perform this.save)}}>
+  <div class="box is-sideless is-fullwidth is-marginless">
+    <MessageError @errorMessage={{this.error}} />
+    {{#each @model.formFields as |field|}}
+      {{#if (eq field.name "issuingCertificates")}}
+        <div class="has-top-margin-m has-bottom-margin-s">
+          <h2 class="title is-4">
+            Issuer URLs
+          </h2>
+        </div>
+      {{/if}}
+      <FormField @attr={{field}} @model={{@model}} @showHelpText={{false}}>
```

Comment:
If we're no longer using openApi, can the `@showHelpText` be removed?

Reply:
Yes for sure. Good catch!